projects/plugins/jetpack/changelog/update-use-assets-enqueue-stats

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: other
+Comment: Updated composer.lock.
+
+
```

Comment:
Should we mention the removed deprecated functions in here?

Reply:
And for that matter, technically the significance on that should be "major" since we're removing methods. Not that we care about significance for Jetpack.

Reply:
👍 8b51e5e30af77a8337420139486cdd3c2c50a65d